TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Probably don't need `DefineConstants`.  

Do you NEED `ProjectGuid`?  Is that just a VS thing?

Probably don't need `DebugSymbols` and `DebugType`, those should just have good defaults.

I think the rest are necessary though, from my spot checking.